### PR TITLE
High Order Visualization

### PR DESCRIPTION
**Pull request by termi-official**
Addresses the second point in #8 via a low order refinement strategy. Basically we create a new dof handler and solution which is the interpolation onto a linear nodal discretization. Partially addressed #8 .

Edit: Maybe we want to move this whole thing into Ferrite, because such an infrastructure is also helpful for other stuff (e.g. LOR preconditioner construction).

### CODE REVIEW

File: docs/src/atopics.md [hunk 1]
```diff
@@ -51,6 +51,35 @@ f
...
+plotter_for = MakiePlotter(dh_for, u_for)
+FerriteViz.surface!(axs[1], plotter_for)
+
+dh,u = manufactured_heat_problem(Triangle, Lagrange{2,RefTetrahedron,5}(), 3)
+dh_for,u_for = for_discretization(dh, u)
+plotter_for = MakiePlotter(dh_for, u_for)
+FerriteViz.surface!(axs[2], plotter_for)
+
+f
+```
```
Comment (koehlerson) [inline docs/src/atopics.md:78]: image
something broken in the documenter deploy for this plot

Comment (termi-official) [reply docs/src/atopics.md:78]: Derp.

Comment (termi-official) [reply docs/src/atopics.md:78]: Fixed.

File: docs/src/atopics.md [hunk 2]
```diff
@@ -51,6 +51,35 @@ f
 
 An alternative to this approach is to compute gradient quantities at samples points and plot these via `arrows`.
 
+### High-order fields
+
+The investigation of high-order fields is currently only supported via a first-order refinment of the
+problem.
```
Comment (koehlerson) [inline docs/src/atopics.md:57]:
```suggestion
The investigation of high-order fields is currently only supported via a first-order refinment of the problem.
```

File: docs/src/atopics.md [hunk 3]
```diff
@@ -51,6 +51,35 @@ f
...
+Here the high-order approximation is replaced by a first order approximation of the field, which is
```
Comment (koehlerson) [inline docs/src/atopics.md:58]:
```suggestion
Here, the high-order approximation is replaced by a first order approximation of the field, which is
```

File: docs/src/index.md [hunk 4]
```diff
@@ -38,3 +38,5 @@ This package offers a set of unique features that are not easily reproducible wi
 - deformed plots available for `solutionplot` and `wireframe`
```
Comment (koehlerson) [inline docs/src/index.md:38]:
```suggestion
- deformed plots available for `solutionplot` and `wireframe` with linear geometry
```

File: docs/src/tutorial.md [hunk 5]
```diff
@@ -96,6 +108,8 @@ WGLMakie.current_figure()
 Note that we can replace the plane withs some other object or a decision function. Such a function takes
 the grid and a cell index as input and returns a boolean which decides whether a cell is visible or not.
 
+### What next?
```
Comment (koehlerson) [inline docs/src/tutorial.md:111]:
```suggestion
### What's next?
```

**Comment by termi-official** [comment]
Ping @koehlerson - something missing?

**Comment by koehlerson** [comment]
Can be merged, how we want to release things? After this new release?

**Comment by termi-official** [comment]
I would say we also merge the fix for the face viz and make a new release?

**Pull request merged**